crates/biome_js_analyze/src/lint/nursery/use_semantic_elements.rs

```diff
@@ -59,25 +63,83 @@ impl Rule for UseSemanticElements {
             let is_not_interative =
                 AriaRoles.is_not_interactive_element(element_name, extract_attributes);
             if is_not_interative {
-                return Some(AnyJsxAttribute::from(attr));
+                let static_value = attr.as_static_value().unwrap();
+                let role_value = static_value.as_string_constant().unwrap();
```

Comment:
Read our contribution guide that explains [how to navigate the CST inside rules](https://github.com/biomejs/biome/blob/main/crates/biome_analyze/CONTRIBUTING.md#navigating-the-cst).